### DISCUSSION TITLE
How to set different control/reference groups?

### DISCUSSION THREAD

**Discussion by maximus-sci** [FrederickHuangLin/ANCOMBC#55] [Q&A] [answered]
Hello,

I'm performing an analysis of microbiome data with ANCOM-BC where samples are taken from 4 different treatments (A->D).

I think that, by default, the first variable is set as the reference and the results are A vs B, A vs C, etc.

I'd like to perform a few different calculations like A vs B, C vs D, etc.
How can I specify these?

**Comment by FrederickHuangLin** [comment, marked as answer]
Hi @maximus-sci,

Suppose you have 4 groups, A, B, C, and D, and you are interested in the comparison of C vs. D. A quick and easy way is probably to subset the data to contain only C and D, and then perform ANCOMBC. A sample code would like

```
sub_data = data %>% filter (group %in% c("C", "D"))
```

Hope it helps,
Huang

**Reply by maximus-sci** [reply]
In other algorithms I've used (like DESeq2 and limma) the total dataset is used to inform the overall variance of each observation (each gene or each OTU). If I did this subsetting with those algorithms, I'd be losing some power to identify difference between groups. Does ANCOMBC work the same way? 

I was thinking of just running ANCOMBC a few different times, setting the reference to be a different variable each time.

**Reply by FrederickHuangLin** [reply]
You are right. Subsetting data leads to loss of power while performing ANCOMBC function multiple times might lead to an increase of FDR. You might want to tighten the FDR control (e.g. set a smaller `alpha` level) in this case. We notice that this is not an ultimate solution and we are evolving ANCOM-BC to more complex designs such as the multi-group comparisons in your case.

Best,
Huang
